Source/Analytics/Concepts/Map/CaseReportId.cs

```diff
@@ -0,0 +1,12 @@
+using Dolittle.Concepts;
+using System;
+namespace Concepts
+{
+    public class CaseReportId : ConceptAs<Guid>
```

Comment:
Is this needed? I don't think it is necessary anywhere else in the system – what is the new need here?